### PR TITLE
Feature - Flagsmith Client

### PR DESCRIPTION
**Pull request by fzia**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+plugins {
+    kotlin("jvm") version "1.6.20"
+    application
+
+    id("org.jlleitschuh.gradle.ktlint") version "10.2.1"
+    id("maven-publish")
+}
+
+group = "com.flagsmith"
+version = "1.0.0"
```
Comment (matthewelwell) [inline build.gradle.kts]: I'm not sure of the conventions with regards to Kotlin / Java packages. Should the Kotlin client follow the version numbers of the Java client, or is it fine to have them totally different?

File: build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+
+dependencies {
+    testImplementation(kotlin("test"))
+
+    // https://mvnrepository.com/artifact/com.flagsmith/flagsmith-java-client
+    api("com.flagsmith:flagsmith-java-client:5.0.0") {
```
Comment (matthewelwell) [inline build.gradle.kts:23]: I guess this client will fail to build until the java client exists in the maven repository?

File: build.gradle.kts (hunk 1 above)
Comment (fzia) [reply build.gradle.kts]: You can do both. But I believe what you are suggesting, to keep them as same as Java, makes more sense.

File: build.gradle.kts (hunk 2 above)
Comment (fzia) [reply build.gradle.kts:23]: Correct. This probably why Github actions won't run until the release has been made for Java client 5.0.0.

File: build.gradle.kts (hunk 1 above)
Comment (matthewelwell) [reply build.gradle.kts]:
```suggestion
version = "5.0.0"
```

**Review by matthewelwell — APPROVED** [review]

**Comment by matthewelwell** [comment]
Despite the failures, I'm going to merge this as it will continue to fail until 5.0.0 of the java client is released.

**Pull request merged**